<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="73.4780" tests="4" failures="1">
  <testsuite name="Root Suite" timestamp="2022-06-02T04:52:12" tests="0" file="cypress/integration/firstTest.spec.js" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="First suite" timestamp="2022-06-02T04:52:12" tests="4" time="73.4540" failures="1">
    <testcase name="First suite check choose date in field" time="0.0000" classname="check choose date in field">
      <failure message="Timed out retrying after 4000ms: expected &apos;&lt;input#search-dateIn-boo.tArrivalDate.hasDatepicker&gt;&apos; to have value &apos;26/05/2022&apos;, but the value was &apos;26/06/2022&apos;" type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: expected '<input#search-dateIn-boo.tArrivalDate.hasDatepicker>' to have value '26/05/2022', but the value was '26/06/2022'
    at Context.eval (https://all.accor.com/__cypress/tests?p=cypress\integration\firstTest.spec.js:112:48)]]></failure>
    </testcase>
    <testcase name="First suite check click with list" time="12.2210" classname="check click with list">
    </testcase>
    <testcase name="First suite check click with list2" time="10.6620" classname="check click with list2">
    </testcase>
    <testcase name="First suite check click with list 3" time="21.2000" classname="check click with list 3">
    </testcase>
  </testsuite>
</testsuites>